osx/qmk_toolbox/AppDelegate.m

```diff
@@ -59,14 +59,25 @@ - (IBAction) flashButtonClick:(id) sender {
             error++;
         }
         if (error == 0) {
-            [_printer print:@"Attempting to flash, please don't remove device" withType:MessageType_Bootloader];
+            if (self.autoFlashButton.state != NSOnState) {
+                self.flashButton.enabled = NO;
+                self.resetButton.enabled = NO;
+                self.clearEEPROMButton.enabled = NO;
```

Comment:
I think it'd be nice to shift these various groups (the enable and disable) of 3 lines to another method since they're being used a bunch here to DRY things up.

Reply:
Whoops, yeah, I forgot to finish that.